protoc-gen-twirpy/main.go

```diff
@@ -128,7 +128,7 @@ func GenerateTwirpFile(fd *descriptor.FileDescriptorProto) (*plugin.CodeGenerato
 func getImportAndObject(name string) (string, string) {
 	name = strings.TrimPrefix(name, ".")
 	paths := strings.Split(name, ".")
-	importPath := strings.Join(paths[:len(paths)-1], ".") + "_pb2"
+	importPath := "." + strings.Join(paths[:len(paths)-1], ".") + "_pb2"
```

Comment:
I think this will break with Verloop's internal setup. There we expect complete path.